remoteworkitem/trackeritem_repository_test.go

```diff
@@ -0,0 +1,30 @@
+package remoteworkitem
+
+import (
+	"testing"
+
+	"github.com/almighty/almighty-core/models"
+	"github.com/almighty/almighty-core/resource"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestConvert(t *testing.T) {
+	resource.Require(t, resource.Database)
+	ts := models.NewGormTransactionSupport(db)
+
+	ts.Begin() // TODO: will have to put this inside transaction.do{}
+	remoteItemData := map[string]string{
+		"content": `{"title":"linking","id":"http://github.com/sbose/api/1","state":"closed","body":"body of issue","user.login":"sbose78","assignee.login":"pranav"}`,
+	}
+	workItem, err := convert(ts, 1, remoteItemData, ProviderGithub)
```

Comment:
@sbose78 You can create tracker & tracker query like this: https://github.com/baijum/almighty-core/blob/b05b397a9486f965ab4c0274886927efa8465b86/remoteworkitem/trackeritem_test.go